src/auro-nav.js

```diff
@@ -235,11 +235,8 @@ export class AuroNav extends LitElement {
     icon.category = category;
     icon.name = name;
 
-    icon.style.width = size;
-    icon.style.height = size;
-    icon.style.lineHeight = `1.5rem`;
+    icon.style = size;
```

Comment:
The param name `size` seems misleading as it can be any style-object.

Reply:
updated